packages/ember-views/lib/views/view.js

```diff
@@ -1305,6 +1305,10 @@ var View = CoreView.extend(
     }
 
     this._super.apply(this, arguments);
+
+    if (!this._viewRegistry) {
```

Comment:
@rwjblue is correct, there are a *ton* of tests (and presumably apps) that create container-less Views. That's what this code is doing.